src/UI/FSMPMenu.cpp

```
@@ -0,0 +1,1578 @@
+	{
+		frac = std::clamp(frac, 0.0f, 1.0f);
+		ImGuiMCP::ImVec2 mn{}, mx{};
+		ImGuiMCP::GetItemRectMin(&mn);
+		ImGuiMCP::GetItemRectMax(&mx);
+		ImGuiMCP::ImVec4 c = kAccent;
+		c.w = 0.25f;  // translucent so the grab and the value text stay readable underneath
+		ImGuiMCP::ImDrawListManager::AddRectFilled(ImGuiMCP::GetWindowDrawList(), mn,
+			ImGuiMCP::ImVec2{ mn.x + (mx.x - mn.x) * frac, mx.y }, ImGuiMCP::GetColorU32(c), 3.0f, 0);
+	}
+
+	std::string readFileBytes(const std::filesystem::path& path)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)